components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,374 @@
+        }
+        const options = [];
+        data.meetings.forEach((meeting) => {
+          meeting.recording_files.forEach((recording) => {
+            const label = `${meeting.topic} - (Format: ${recording.file_type}) (From ${recording.recording_start} to ${recording.recording_end})`;
+            options.push({
+              label,
+              value: {
+                meetingId: meeting.id,
+                label,
+                value: recording.id,
+              },
+            });
+          });
+        });
+
+        return {
+          options,
+          context: {
+            nextPageToken: data.next_page_token,
+          },
+        };
+      },
+    },
+    meeting: {
+      type: "string",
+      label: "Meeting",
+      description: "The meeting ID",
+      async options({
+        prevContext,
+        page,
+      }) {
+        if (!prevContext.nextPageToken && page > 0) {
+          return [];
+        }
+        const data = await this.listMeetings(prevContext.nextPageToken);
+        return {
+          options: data?.meetings.map((meeting) => ({
+            label: meeting.topic,
+            value: {
+              label: meeting.topic,
+              value: meeting.id,
+            },
+          })),
+          context: {
+            nextPageToken: data.next_page_token,
+          },
+        };
+      },
+    },
+    panelist: {
+      type: "string",
+      label: "Panelist",
+      description: "The panelist ID or panelist email",
+      async options({ webinar }) {
+        if (!webinar) {
+          return [];
+        }
+        try {
+          const data = await this.listWebinarPanelists(get(webinar, "value", webinar));
+          return data?.panelists.map((panelist) => ({
+            label: `${panelist.name} <${panelist.email}>`,
+            value: {
+              label: panelist.name,
+              value: panelist.id,
+            },
+          }));
+        } catch {
+          return [];
+        }
+      },
+    },
+    occurrenceId: {
+      type: "string",
+      label: "Occurrence ID",
+      description: "Provide this field to view meeting details of a particular occurrence of the [recurring meeting](https://support.zoom.us/hc/en-us/articles/214973206-Scheduling-Recurring-Meetings).",
+      optional: true,
+      async options({
+        meeting,
+        isWebinar,
+      }) {
+        if (!meeting) {
+          return [];
+        }
+        const occurrences = await this.listMeetingsOccurrences(
+          get(meeting, "value", meeting),
+          isWebinar,
+        );
+        return occurrences.map((occurrence) => ({
+          label: `${occurrence.start_time} (${occurrence.status})`,
+          value: occurrence.occurrence_id,
+        }));
+      },
+    },
+    registrants: {
+      type: "string[]",
+      label: "Registrants",
+      description: "The meeting registrant. If you disable the \"Structured Mode\", please provide an array of objects with `id` and `email`",
```

Comment:
Can you put like a practical example for users to just be able to copy paste, something like
```suggestion
      description: "The meeting registrant. If you disable the \"Structured Mode\", please provide an array of objects with `id` and `email`. (Eg. `{{ [ { id: 123, email: \"123@mail.com\" } ] }}`)",
```